tests/functional_tests/launch_test/launch_test.py

```diff
@@ -33,4 +35,27 @@ async def test_run_bot():
     bot.task_manager.init_async_loop()
     await start_bot(bot, init_logger())
     await asyncio.sleep(10)
-    bot.task_manager.stop_tasks()
```

Comment:
Why don't we use `stop_tasks()` anymore ?

Reply:
to be able to await the stop (stop tasks is not a coroutine and is using call_soon which makes it not awaitable)

Reply:
Why not just calling `task_manager.stop_tasks` in `async def stop_bot(bot)`  ?